webapp/app/css/perf.css

```diff
@@ -143,4 +143,8 @@ to { opacity: 0; }
 #scrollable-dropdown-menu .tt-dropdown-menu {
   max-height: 150px;
   overflow-y: auto;
-}
\ No newline at end of file
+}
+#showHideCheck{
```

Comment:
Also this should be a class, not an id. Also, put it below .graph-legend and make the property only apply in its context:

#show-hide-check -> .graph-legend .show-hide-check.